docs/azure/dependency-injection.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Dependency injection with the Azure SDK for .NET
+description: Learn how to use dependency injection with the Azure SDK for .NET client libraries
+ms.date: 05/12/2021
+ms.custom: devx-track-dotnet
```

Comment:
```suggestion
```

Why is this needed? This is in the .NET docs, not the Azure repo - it's my understanding that the `devx-track-*` metadata is only for the Azure docs.

Reply:
oh I added because other docs seem to have that as well. I'll remove it if it's not needed.

Reply:
Confirmed that it's not needed.